src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -59,7 +59,7 @@ public class CustomAsyncResult : IAsyncResult
     public static class Helpers
     {
         public static bool IsWindowsImplementation { get; } = PlatformDetection.IsWindows && PlatformDetection.IsNotOneCoreUAP;
-        public static bool IsNotWindowsImplementation => !IsWindowsImplementation;
+        public static bool IsNotWindowsImplementation => !IsWindowsImplementation && PlatformDetection.IsNotOneCoreUAP;
```

Comment:
>Not in CI, but these tests may be used elsewhere?

We can use UWP (uap or uap-aot) tests on our dev boxes using the right msbuild switch

By the way, many of the uses of the IsNotOneCoreUAP is wrong.  It needs to be switches to the new PlatformDetection.IsUap flag I added.